rebuild.sh

```diff
@@ -0,0 +1,12 @@
+# dont forget submodules
```

Comment:
please remove

Reply:
Done

Reply:
you must `git rm` this file and commit to actually remove it

Reply:
also delete that binary file `prime_testd`